testing/qt5-qtserialport/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Denis Ryabyy <vv1r0x@gmail.com> 
+# Maintainer: Denis Ryabyy <vv1r0x@gmail.com>
+pkgname=qt5-qtserialport
+pkgver=5.6.0
+pkgrel=1
```

Comment:
I've increased it every time when I test build. Now I know how to do it right way, so fixed.